bot/handlers/administration/remove_subscription_handler.py

```diff
@@ -10,18 +14,26 @@
     get_no_username_provided_message,
     get_subscription_removed_message,
 )
+from bot.utils.functions import validate_argument_count
 
 
 class RemoveSubscriptionHandler(BotMessageHandler):
     def get_commands(self) -> List[str]:
         return ["removesubscription", "rmsub"]
 
-    async def _do_handle(self, message: Message) -> None:
-        content = message.text.split()
-        if len(content) < 2:
-            return await self._reply_invalid_args_count(message, get_no_username_provided_message())
+    def _get_validator_functions(self) -> List[Callable[[Message], Awaitable[bool]]]:
+        return [
+            self._validate_argument_count,
+        ]
 
-        username = content[1]
+    async def _validate_argument_count(self, message: Message) -> bool:
+        return await validate_argument_count(
+            message, 2, self._reply_invalid_args_count,
+            get_no_username_provided_message(),
+        )
```

Comment:
w sumie na to możesz w ogóle robić lambdę zamiast robić specjalną funkcję, ale przejrzyj sobie czy wygodniej, luźna sugestia, nie wiem co będzie lepsze, jak uważasz xD

Reply:
![obraz](https://github.com/user-attachments/assets/731a2da1-5d21-4f14-af19-c31a668b0a6b)
że tutaj od razu lambda ? nie wiem wydaje mi się że lepiej jak jest spójnie bo żeby tu były jednolicie calle funkcji